src/reduced_basis/rb_construction_base.C

```diff
@@ -398,9 +398,9 @@ void RBConstructionBase<Base>::load_training_set(const std::map<std::string, std
         }
 
       // Ensure that the parameters are the same.
-      for (auto & [param_name, _] : _training_parameters)
+      for (auto & it : _training_parameters)
```

Comment:
I really prefer not to use the name `it` here, since, elsewhere in the code, that usually refers to an iterator variable, whereas in range-based for-loops like this, it is a reference to a `std::pair`, hence the original variable name `pr`.